### DISCUSSION TITLE
RSE-AUNZ Community - What does the community want out of RSE-AUNZ membership? (Session 2 - Main Room)

### DISCUSSION THREAD

**Discussion by rowlandm** [RSEAA/RSEAA.github.io#131] [Discussion topics for #RSEAA24]
I went through ~150 member requests for the RSE-AUNZ google group and put together a spreadsheet that has a very rough view of what members wanted to get out of the community.

Here are the top responses

Generic Comment | count | Personal comment
-- | -- | --
Looking for jobs | 17 | We are sharing jobs as part of the regular digest we send before events
Connecting with other RSEs | 16 | We do provide opportunities for synchronous but not asynchronous communication. RSEAA24 has a Slack workspace now that could potentially help .
Stay up to date | 13 | We do provide updates as part of the regular digest we send before events. Unsure if these updates are appropriate to the community though  (ie. tech news, policy development, skills ?).
Aligns with my job | 12 |  That shows they identify as RSEs.
Looking for RSEs to hire | 3 | How can we connect hiring managers with RSEs both proactively and reactively?
Recently started a job that this aligns with | 3 | That shows they identify as RSEs.
Share knowledge and experience | 3 | Don't always do this well proactively.
Leadership in the sector / Contribute to the Community | 2 | How can we imrpove recruitment and retention of volunteers who want to contribute back?

**Comment by rowlandm** [comment]
[RSEAA24 Live Polling results.pdf](https://github.com/user-attachments/files/16972397/RSEAA24.Live.Polling.results.pdf)

**Reply by rowlandm** [reply]
[RSEAA24-051 Breakdown of Live Polling Qualitative review](https://docs.google.com/spreadsheets/d/170UamNZVfyaZrGR1s7T-t9v9XGVn6kKuvqgoBkikcg8/edit?gid=470977273#gid=470977273)

**Comment by rowlandm** [comment]
- Lot of confusion around labels, struggle to define myself. If it is confusing for us, what about the managers!
- What does the high level manager thinking of when pulling together a team? Vague and explanation needed. Education and outreach for building teams.
- Is that because we are generalists?
- The table of top responses matches what I've heard at other forums,  jobs (finding and hiring), connecting with others

**Comment by rowlandm** [comment]
![Screenshot 2024-09-11 at 3 51 45 pm](https://github.com/user-attachments/assets/0b46f67a-68d6-4aea-8eb8-d8fa57203830)

Training Capability Maturity Model from the [Research Software Engineer (RSE) Leadership Training Course Outline](https://figshare.com/articles/presentation/Research_Software_Engineer_RSE_Leadership_Training_Course_Outline/22801898)


- Do I need a postdoc or a walking talkiing microscope
- Similarly, HPC paid first with no sysadmins - then added training - what is the total cost of research.

**Reply by robmoss** [reply]
I'm preparing an "Introduction to Debugging" workshop for an upcoming national meeting, because we have RHD students and postdocs for whom writing code is a substantial part of their research, but many of them do not make use of debuggers when trying to identify and fix errors in their code. In one of our recent community of practice meetings, many participants expressed a strong interest in having workshop on this topic, and our high-level pitch focuses on spending less time trying to fix your code and spend more time doing your actual research.

**Comment by rowlandm** [comment]
Can we raise awareness of "Total Cost of Research"

**Reply by rowlandm** [reply]
Education and outreach for people writing grants and people reviewing grants was brought up.

**Comment by rowlandm** [comment]
- AAF wants to work with researchers to understand what they want and do it collaboratively.
- Create a message and tailor it for each domain.

**Comment by rowlandm** [comment]
Building pipes between systems including auth and security and privacy.

**Comment by rowlandm** [comment]
- cooperate with the Carpentries

**Comment by rowlandm** [comment]
- The main issue is the invisibility of the RSE community. They don't unite well. We should get together and be visible to the academia and the funders!

**Reply by rowlandm** [reply]
- This could be international in scope!

**Comment by rowlandm** [comment]
- SOmeone was hired as an RSE!

**Reply by rowlandm** [reply]
Within a domain

**Reply by rowlandm** [reply]
ANU is the university

**Reply by rowlandm** [reply]
Once you are labelled RSE you have to go through specific questions -> what is XYZ bash command?

**Reply by rowlandm** [reply]
Moving from invisibility to being pidgen holed.

**Reply by rowlandm** [reply]
ACCESS-NRI has RSE positions.

**Comment by rowlandm** [comment]
- good that RSE has recognition but it doesn't have the graduatiion and levels
- eg. spectrum of RSEs and seniority

**Reply by rowlandm** [reply]
- How mixed professional type roles get funded in research projects

**Comment by rowlandm** [comment]
Hearing about definitions of RSE roles - here are the roles definitions (drafted by founding director 
[@James Smithies](https://rse-asia-australia-24.slack.com/team/U07LL41KHQD)
) that inform our job descriptions at King’s Digital Lab in case anyone is interested http://doi.org/10.5281/zenodo.2559235
